source/community/release_notes/redwood.rst

```diff
@@ -13,9 +13,65 @@ These are the release notes for the Redwood release, the 18th community release
+   ``JWT_AUTH['JWT_PRIVATE_SIGNING_JWK']`` Django setting in LMS
+   contains the full complement of private key ``numbers.*``
 
-Learner Experiences
-*******************
+   -  Background: In LMS, we switched from the pyjwkest
+      library to PyJWT for signing JWTs. (pyjwkest is now unmaintained.)
+      However, PyJWT has stricter requirements for the private key in
+      ``JWT_PRIVATE_SIGNING_JWK``. Before you upgrade to Redwood, you
+      will need to update this key using a script. Otherwise, JWT
+      signing will fail, and users will be unable to log in.
+
+   -  Steps:
+
+      1. Locate ``JWT_PRIVATE_SIGNING_JWK`` in your deployment
+         configuration.
+
+      2. Check if the JSON contains all of the following params: ``p``,
+         ``q``, ``dp``, ``dq``, and ``qi``. If it does, you don’t need
+         to do anything further. Otherwise, continue.
+
+      3. In your edx-platform virtualenv, run
+         ``python3 scripts/jwk-precompute-params.py`` and follow the
+         prompts. (It will ask you to paste in the current JSON.)
+
+      4. Update your config’s ``JWT_AUTH['JWT_PRIVATE_SIGNING_JWK']``
+         with the output of the script.
+
+      5. You may wish to compare the contents of the private key before
+         and after running the script, and verify that the only changes
+         it has made to the contents of the JSON are that the ``p``,
+         ``q``, ``dp``, ``dq``, and ``qi`` params have been added. (Some
+         or all may already have been present.)
+
+   -  Notes:
+
+      1. This should be done while you are still running Quince—it is
+         safe to do ahead of the upgrade, and should not have any
+         visible effect at that time.
+
+      2. This key must be handled very carefully. Anyone in possession
+         of the key may impersonate users.
+
+- `studio_home.enable_global_staff_optimization flag no longer works in Studio MFE <https://github.com/openedx/wg-build-test-release/issues/380>`_
+   - Background: This flag works for the legacy Studio Home, improving performance by
+     adding an organization search bar.  This flag does not work in the Studio Home MFE, as it
+     is not needed within the MFE. The MFE already have a search bar and pagination, which
+     improves performance for those home pages with lots of courses.
+     adding an organization search bar. But as the report mentions, it's not working
+     correctly in the Studio Home MFE. The good thing is that this flag is not needed
+     in the Studio Home MFE since we already have a search bar and pagination, which
+     improves performance for those home pages with lots of courses. I wouldn't consider
+     it a "breaking change" but a known issue for those who want to turn on the Studio
+     Home MFE and are using the flag.
+
+Learner & Instructor Experiences
+********************************
+
+See the Product Release Notes.
+******************************
 
 
 Instructor Experiences
```

Comment:
```suggestion
```

Reply:
Also need to remove the underline one line below (I can't make a comment it's too far down in the diff)

Reply:
Did that in next revision. Thanks!